### DISCUSSION TITLE
Logout a specific user

### DISCUSSION THREAD

**Discussion by florincretzan** [codeigniter4/shield#561] [Q&A]
Hi guys! Is there a way you can logout a specific user ? I want to be able to logout ( from the admin backend ) a user with a specific id. Thank you !

**Comment by kenjis** [comment]
It seems there is no easy way.
Why do you want to make a specific user logout?

Banning users is on going.
See #559

**Reply by florincretzan** [reply]
Just curious for some specific use case scenario. Thank you !
